pkg/network/ebpf/c/tracer.c

```diff
@@ -286,7 +286,7 @@ int BPF_BYPASSABLE_KPROBE(kprobe__tcp_close, struct sock *sk) {
     // upsert the timestamp to the map and delete if it already exists, flush connection otherwise
     // skip EEXIST errors for telemetry since it is an expected error
     __u64 timestamp = bpf_ktime_get_ns();
-    if (bpf_map_update_with_telemetry(conn_close_flushed, &t, &timestamp, BPF_NOEXIST, -EEXIST) == 0) {
+    if (bpf_map_update_with_telemetry(conn_close_flushed, &t, &timestamp, BPF_NOEXIST, -EEXIST, -E2BIG) == 0) {
```

Comment:
I am not sure I follow the reasoning for suppressing E2BIG errors. 
Error suppression should only be used for errors we cannot do anything about, or are expected due to ABI usage. E2BIG does not fall into this category imo.

Reply:
+1, I think we should not suppress E2BIG

Reply:
basically this map is used as a cache for closed connection flushes. if the cache fills up, we will not clean up closed connections (and some other maps will leak). I am fine with focusing on making the cache not fill up in the first place instead, but as a backstop I think it would have been useful to allow this codepath to flush connection closures.